cli/rbd/trash/trash.py

```diff
@@ -0,0 +1,240 @@
+from copy import deepcopy
```

Comment:
Does trash folder is necessary?

Reply:
Yes because again this is just a partial PR. We have next one for trash purge schedule operations where we will require folder.